src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -1074,6 +1074,25 @@ export default class DefaultDeviceController
     return outputNode.stream;
   }
 
+  static synthesizeStereoTones(toneHzLeft: number, toneHzRight: number): MediaStream {
```

Comment:
Can we add a doc on what this method do?

Reply:
Sure. Where would be a good place to add that?

Reply:
You can just add it here in this method. For instance methods, we document in the interface level but for static we don't have one standard place yet. @simmkyu

Reply:
Also is this API only used for testing? If so should it be here or should it just be in the demo app.

Reply:
Added comment in latest push and moved into `meetingV2.ts` as its only used for testing